scripts/setup_ebpf_cicd_stress_tests.ps1

```diff
@@ -2,6 +2,8 @@
 # SPDX-License-Identifier: MIT
 
 param ([parameter(Mandatory=$false)][string] $Target = "TEST_VM",
+       [parameter(Mandatory=$false)][bool] $KmTracing = $false,
+       [parameter(Mandatory=$false)][string] $KmTraceType = "file",
```

Comment:
Given the goal of the PR, this should probably not be the "file", or the new code WPR profile won't be used.

Separately, I wonder if we couldn't just use this new prifle in place of the old one for all jobs, given that the driver job tests don't run for too long.

Reply:
Where possible, we prefer file based traces, as no traces are lost.  We switch to in-memory tracing only for those tests that have the potential to created trace files of un-bounded size.

The .wprp file now has profiles for both modes, and the applicable profile is selected based on the `KmTraceMode` value